ert/data/__init__.py

```diff
@@ -1,19 +1,26 @@
+from ert.data._record import RecordType
 from ert.data._record import Record
+from ert.data._record import NumericalRecord
+from ert.data._record import BlobRecord
+from ert.data._record import make_record
 from ert.data._record import EnsembleRecord
 from ert.data._record import MultiEnsembleRecord
 from ert.data._record import RecordTransmitter
 from ert.data._record import SharedDiskRecordTransmitter
 from ert.data._record import InMemoryRecordTransmitter
-from ert.data._record import RecordType
```

Comment:
I did not find the order logical, and moved the record type closer to the types of the records.  Do you have a particular ordering in mind? (i.e. alphabetically?)

Reply:
Nope, I just think that unless we have an agreed upon order of imports we not reorder. If we all reorder after personal preference, we'll all be shuffling imports :) It might even be that we could get `pylint` or `flake8` to impose an order.

I'm not particularly opinionated about the order itself, just about the frequent reshuffling and the noice it induces.

Reply:
well, it is easily undone.